crates/subspace-node/src/commands/run.rs

```diff
@@ -307,17 +309,30 @@ pub async fn run(run_options: RunOptions) -> Result<(), Error> {
                             }
                         };
 
+                        let consensus_chain_sync_params =
+                            snap_sync_orchestrator.map(|snap_sync_orchestrator| {
+                                ConsensusChainSyncParams {
+                                    snap_sync_orchestrator,
+                                    fork_id,
+                                    network_service: consensus_chain_network_service,
+                                    sync_service: consensus_chain_sync_service,
+                                    backend: consensus_chain_node.backend.clone(),
+                                    subspace_link,
```

Comment:
`subspace_link` is a low-level-ish thing that should not be available to domains. If you need a specific stream then make it an argument of the domains and send it here. No need to send the whole `subspace_link` and